jetty-core/jetty-compression/jetty-compression-server/src/main/java/org/eclipse/jetty/compression/server/CompressionConfig.java

```diff
@@ -110,72 +110,72 @@ public static Builder builder()
-        return Collections.unmodifiableSet(includes);
+        Set<String> excluded = compressMimeTypes.getExcluded();
+        return Collections.unmodifiableSet(excluded);
     }
 
     /**
-     * Get the set of excluded MIME types for Response compression.
+     * Get the set of excluded Path Specs for response compression.
      *
-     * @return the set of excluded MIME types
-     * @see #getCompressMimeTypeIncludes()
+     * @return the set of excluded Path Specs
+     * @see #getCompressPathIncludes()
      */
-    @ManagedAttribute("Set of Mime-Types Excluded from Response compression")
-    public Set<String> getCompressMimeTypeExcludes()
+    @ManagedAttribute("Set of Response Compression Path Exclusions")
+    public Set<String> getCompressExcludePaths()
     {
-        Set<String> excluded = compressMimeTypes.getExcluded();
+        Set<String> excluded = compressPaths.getExcluded();
         return Collections.unmodifiableSet(excluded);
     }
 
     /**
-     * Get the set of included MIME types for Response compression.
+     * Get the set of included HTTP methods for Response compression
      *
-     * @return the filter list of included MIME types
-     * @see #getCompressMimeTypeExcludes()
+     * @return the set of included HTTP methods
+     * @see #getCompressExcludeMethods()
      */
-    @ManagedAttribute("Set of Mime-Types Included in Response compression")
-    public Set<String> getCompressMimeTypeIncludes()
+    @ManagedAttribute("Set of HTTP Method Inclusions")
+    public Set<String> getCompressIncludeMethods()
     {
-        Set<String> includes = compressMimeTypes.getIncluded();
+        Set<String> includes = compressMethods.getIncluded();
         return Collections.unmodifiableSet(includes);
     }
 
     /**
-     * Get the set of excluded Path Specs for response compression.
+     * Get the set of included MIME types for Response compression.
      *
-     * @return the set of excluded Path Specs
-     * @see #getCompressPathIncludes()
+     * @return the filter list of included MIME types
+     * @see #getCompressExcludeMimeTypes()
      */
-    @ManagedAttribute("Set of Response Compression Path Exclusions")
-    public Set<String> getCompressPathExcludes()
+    @ManagedAttribute("Set of Mime-Types Included in Response compression")
+    public Set<String> getCompressIncludeMimeTypes()
     {
-        Set<String> excluded = compressPaths.getExcluded();
-        return Collections.unmodifiableSet(excluded);
+        Set<String> includes = compressMimeTypes.getIncluded();
+        return Collections.unmodifiableSet(includes);
     }
 
     /**
      * Get the set of included Path Specs for response compression.
      *
      * @return the set of included Path Specs
-     * @see #getCompressPathExcludes()
+     * @see #getCompressExcludePaths()
      */
     @ManagedAttribute("Set of Response Compression Path Exclusions")
     public Set<String> getCompressPathIncludes()
```

Comment:
Rename this to `getCompressIncludePaths()` to be symmetric with the exclude equivalent.